EventsExpress/ClientApp/src/services/EventsExpressService.js

```diff
@@ -289,18 +290,54 @@ export default class EventsExpressService {
             method: "get",
             headers: new Headers({
                 'Content-Type': 'application/json',
-                'Authorization': `Bearer ${localStorage.getItem('token')}`
+                'Authorization': 'Bearer ' + localStorage.getItem('token')
             }),
         });
 
-        return !res.ok
-            ? {
+        if (res.ok) {
```

Comment:
Lines 297 through 326 are duplicated in three methods. Consider extracting those into separate method and reusing it.

Reply:
refactored